rpc/swagger/swagger.yaml

```diff
@@ -1261,6 +1261,9 @@ definitions:
         $ref: "#/definitions/BlockId"
       header:
         $ref: "#/definitions/BlockHeader"
+      number_txs:
+        type: number
+        x-example: 0
```

Comment:
what x- prefix does? can't find it in the docs https://swagger.io/docs/specification/adding-examples/